### DISCUSSION TITLE
Broken download link for v 0.41 snap

### DISCUSSION THREAD

**Discussion by dferri** [anyproto/contributors#207] [Anytype apps]
The link to download the latest snap (0.41.0: https://anytype-release.fra1.cdn.digitaloceanspaces.com/anytype_0.41.0_amd64.snap) from the main download page (https://download.anytype.io/) is broken and returns a 403 with the following content:
```xml
<Error>
<Code>AccessDenied</Code>
<Message/>
<BucketName>anytype-release</BucketName>
<RequestId>tx00000ad3ffcc252d9569f-00667ad203-fe1ae25a-fra1b</RequestId>
<HostId>fe1ae25a-fra1b-fra1-zg02</HostId>
</Error>
```

I apologize if this is not the right place for this issue!

**Comment by filip-anytype** [comment]
Thank you for bringing this to our attention. We'll remove the snap download until it's fixed.
